README.md

```diff
@@ -17,7 +17,9 @@
   - `pip3 install --user awscli`
   - `aws configure`
 
-  When prompted, enter "Access Key ID" and "Secret Access Key" provided by an AWS administrator. See "Troubleshooting" section below if you can't get `aws configure` to run
+  When prompted, enter "Access Key ID" and "Secret Access Key" provided by an AWS administrator. See "Troubleshooting" section below if you can't get `aws configure` to run.
+  - enter 'us-east-1' for region 
+  - hit the enter/return key when prompted to enter a region name 
```

Comment:
Is it intentional that both of these lines are about the region name? When I ran `aws configure`, I was prompted four times:

```
AWS Access Key ID [********************]: 
AWS Secret Access Key [********************]: 
Default region name [us-east-1]: 
Default output format [None]: 
```

Was the last line supposed to be about the default output format?